src/components/menu-app-bar/index.js

```diff
@@ -118,6 +126,15 @@ class MenuAppBar extends React.Component {
   render() {
     const { classes } = this.props
     const { open } = this.state
+    const { auth } = this.props
+
+    const authLinks = auth.uid ? (
+      <SignedInLinks
+        signOutAndHandleDrawerClose={this.signOutAndHandleDrawerClose}
+      />
+    ) : (
+      <SignedOutLinks handleDrawerClose={this.handleDrawerClose} />
+    )
```

Comment:
```js
renderAuthListItems = () => {
  const { auth } = this.props
  return auth.uid 
	? <SignedInListItems
        signOutAndHandleDrawerClose={this.signOutAndHandleDrawerClose}
      />
    : <SignedOutListItems handleDrawerClose={this.handleDrawerClose} />
}
```

Reply:
Also, there is no need to put them in separate components since they are pretty simple and probably not going to be reused... I'd put them in `this.render...()` calls. Up to you though